### ISSUE TITLE
I2P Service launches on non-AppDataLocal nor ProgramData instance

### ISSUE THREAD

**Issue by RowanSkie** [i2p/i2p.i2p#85]
I've reinstalled I2P after letting it decay for a long while. However, once I let it run and then install the service via the router console, I2P runs it's system (through the I2P service) on the following address:
`C:\WINDOWS\system32\config\systemprofile\AppData\Local\i2p`

I now have basically three versions of the I2P directory via that link, my %LOCALAPPDATA% link, and the %PROGRAMDATA% link.

**Comment by eyedeekay** [comment]
Yeah that is awkward at best, I'll have to take a look at this code-wise but I probably need to know a few things on my end. First, I will need to know the standard stuff about your system, which Windows version, which Java version, is there anything different about how it's running like is it in a VM, and most importantly which I2P version or versions have you installed?

In the meantime, I'd like to help you resolve those config directories down so that the "config you want" becomes the "config you're using" we can do away with the other two. Am I to understand that ` C:\WINDOWS\system32\config\systemprofile\AppData\Local\i2p` is the directory where changes to the running config are being stored?

**Comment by RowanSkie** [comment]
Oh, whoops, forgot the basic necessities.

Windows 11 64bit 23H2
Java 22
I2L 2.7.0-0

Also, yeah, it is currently using that. Right now as a temporary way to fix things (as well as not lose progress on the non-service version aka. %APPDATA% directory), I created a symlink.

I feel like that my upgrade from pre-1.0 alongside reinstalling with a clean slate (without deleting prior versions) prevented the app from noticing my %PROGRAMDATA% link, instead using the global one.

**Comment by RowanSkie** [comment]
Oh, whoops, forgot the basic necessities.

Windows 11 64bit 23H2
Java 22
I2L 2.7.0-0

Also, yeah, it is currently using that. Right now as a temporary way to fix things (as well as not lose progress on the non-service version aka. %APPDATA% directory), I created a symlink.

I feel like that my upgrade from pre-1.0 alongside reinstalling with a clean slate (without deleting prior versions) prevented the app from noticing my %PROGRAMDATA% link, instead using the global one.
